Add Chef iQ integration (#46301)
home-assistant/home-assistant.github.io: source/_integrations/chefiq.markdown

## Sensors

A probe exposes the following sensors:

- **Food temperature**: the food (internal) temperature at the probe tip.
- **Ambient temperature**: the ambient (cooking environment) temperature.
- **Probe tip 1-4 temperature**: the temperatures measured along the probe shaft.
- **Battery**: the probe battery percentage.
- **SoC temperature**: the probe's internal electronics temperature (diagnostic).
- **Signal strength**: the Bluetooth signal strength (diagnostic, disabled by default).

Battery and SoC temperature are published in a separate advertisement from the temperatures, so they can take a few seconds longer to appear after setup.

## Known limitations

The integration is passive and read-only. It reports the probe's broadcast readings, but cannot start cooks, set target temperatures, or change probe settings. Those features require the Chef iQ app.

## Troubleshooting

If the device is not discovered, make sure the probe is awake and broadcasting by taking it off its dock, that your Bluetooth adapter or proxy is in range of the probe, and that the [Bluetooth](/integrations/bluetooth/) integration is set up and working.

> {% include integrations/remove_device_service.md %}

Author: Addressed

---
title: Chef iQ
description: Instructions on how to integrate Chef iQ wireless probes with Home Assistant.
ha_category:
  - Sensor
ha_release: 2026.7
ha_iot_class: Local Push
ha_config_flow: true
ha_codeowners:
  - '@Invader444'
ha_domain: chefiq
ha_integration_type: device
ha_platforms:
  - diagnostics
  - sensor
ha_bluetooth: true
ha_quality_scale: bronze
---

The **Chef iQ** {% term integration %} lets you read the temperatures from Chef iQ wireless cooking probes directly over Bluetooth Low Energy. No cloud account, base station, or hub is required: the probe broadcasts its readings and Home Assistant listens passively.

> {% include integrations/config_flow.md %}

Reviewer: ```suggestion
```

Author: Done :+1:

## Supported devices

The following Chef iQ wireless probes are supported:

- Chef iQ Smart Wireless Meat Thermometer (CQ60)
- Chef iQ Smart Wireless Meat Thermometer (CQ50)

> The **iQ Sense** base/hub broadcasts under the same Bluetooth identifier, but it is a Wi-Fi appliance rather than a passive probe, so this integration does not set it up.

Author: Addressed

## Prerequisites

> This integration requires a working [Bluetooth](/integrations/bluetooth/) setup, either a local Bluetooth adapter or an [ESPHome Bluetooth proxy](/integrations/esphome/). The probe is a sleepy device that only broadcasts while it is awake, so make sure it is awake (for example, by taking it off its charging dock) before adding it.

Author: Addressed

## Sensors

A probe exposes the following sensors:

- **Food temperature**: the food (internal) temperature at the probe tip.
- **Ambient temperature**: the ambient (cooking environment) temperature.

> - **Probe tip 1-4 temperature**: the temperatures measured along the probe shaft.

Author: Addressed

- **Battery**: the probe battery percentage.

> - **SoC temperature**: the temperature of the probe's internal electronics, the system-on-chip (SoC) (diagnostic).

Author: Addressed

> - **SoC temperature**: the temperature of the probe's internal electronics, or system-on-chip (diagnostic).

Author: Addressed

- **Signal strength**: the Bluetooth signal strength (diagnostic, disabled by default).

> Battery and SoC temperature are published in a separate advertisement from the temperatures, so they can take a few seconds longer to appear after setup.

Author: Addressed

> Battery and SoC temperature are published in a separate advertisement from the temperatures, so they can take a few seconds longer to appear after setup.

Author: Addressed

## Known limitations

The integration is passive and read-only. It reports the probe's broadcast readings, but cannot start cooks, set target temperatures, or change probe settings. Those features require the Chef iQ app.

## Troubleshooting

> If the device is not discovered, make sure the probe is awake and broadcasting by taking it off its dock, that your Bluetooth adapter or proxy is in range of the probe, and that the [Bluetooth](/integrations/bluetooth/) integration is set up and working.

Author: Addressed

> If the device is not discovered, make sure that the probe is awake and broadcasting (take it off its dock), that your Bluetooth adapter or proxy is in range of the probe, and that the [Bluetooth](/integrations/bluetooth/) integration is set up and working.

Reviewer: ```suggestion
If the device is not discovered, try the following:

1. Make sure that the probe is awake and broadcasting (take it off its dock).
2. Make sure your Bluetooth adapter or proxy is in range of the probe.
3. Make sure the [Bluetooth](/integrations/bluetooth/) integration is set up and working.
```

Author: Done :+1:

ha_codeowners:
  - '@Invader444'
ha_domain: chefiq
ha_integration_type: device
ha_platforms:
  - diagnostics
  - sensor
ha_bluetooth: true
ha_quality_scale: bronze
---

The **Chef iQ** {% term integration %} lets you read the temperatures from Chef iQ wireless cooking probes directly over Bluetooth Low Energy. No cloud account, base station, or hub is required: the probe broadcasts its readings and Home Assistant listens passively.

## Supported devices

The following Chef iQ wireless probes are supported:

- Chef iQ Smart Wireless Meat Thermometer (CQ60)
- Chef iQ Smart Wireless Meat Thermometer (CQ50)

The **iQ Sense** base station broadcasts under the same Bluetooth identifier, but it is a Wi-Fi appliance rather than a passive probe and is not supported by this integration.

## Prerequisites

This integration requires a working [Bluetooth](/integrations/bluetooth/) setup, either a local Bluetooth adapter or an [ESPHome Bluetooth proxy](/integrations/esphome/). The probe is battery-powered and only broadcasts while it is awake, so make sure it is awake (for example, by taking it off its charging dock) before adding it.

{% include integrations/config_flow.md %}

> ## Sensors

Reviewer: ```suggestion
## Supported functionality

The **Chef iQ** {% term integration %} provides the following entities.

### Sensors
```

A probe exposes the following sensors:

- **Food temperature**: the food (internal) temperature at the probe tip.
- **Ambient temperature**: the ambient (cooking environment) temperature.
- **Probe tip 1 through 4 temperatures**: the temperatures measured along the probe shaft.
- **Battery**: the probe battery percentage.

> - **SoC temperature**: the temperature of the probe's internal electronics, or system-on-chip (diagnostic).

Author: Addressed

- **Signal strength**: the Bluetooth signal strength (diagnostic, disabled by default).

> Battery and SoC temperature are broadcast in a separate Bluetooth advertisement from the temperature readings, so they can take a few seconds longer to appear after setup.

Author: Addressed

  - '@Invader444'
ha_domain: chefiq
ha_integration_type: device
ha_platforms:
  - diagnostics
  - sensor
ha_bluetooth: true
ha_quality_scale: bronze
---

The **Chef iQ** {% term integration %} lets you read the temperatures from Chef iQ wireless cooking probes directly over Bluetooth Low Energy. No cloud account, base station, or hub is required: the probe broadcasts its readings and Home Assistant listens passively.

## Supported devices

The following Chef iQ wireless probes are supported:

- Chef iQ Smart Wireless Meat Thermometer (CQ60)
- Chef iQ Smart Wireless Meat Thermometer (CQ50)

The **iQ Sense** base station broadcasts under the same Bluetooth identifier, but it is a Wi-Fi appliance rather than a passive probe and is not supported by this integration.

## Prerequisites

This integration requires a working [Bluetooth](/integrations/bluetooth/) setup, either a local Bluetooth adapter or an [ESPHome Bluetooth proxy](/integrations/esphome/). The probe is battery-powered and only broadcasts while it is awake, so make sure it is awake (for example, by taking it off its charging dock) before adding it.

{% include integrations/config_flow.md %}
## Supported functionality

> The **Chef iQ** {% term integration %} provides the following entities.

Reviewer: ```suggestion
The **Chef iQ** {% term integration %} provides the following {% term entities %}.
```

### Sensors

A probe exposes the following sensors:

- **Food temperature**: the food (internal) temperature at the probe tip.
- **Ambient temperature**: the ambient (cooking environment) temperature.
- **Probe tip 1 through 4 temperatures**: the temperatures measured along the probe shaft.
- **Battery**: the probe battery percentage.
- **SoC temperature**: the temperature of the probe's system-on-chip (SoC), its internal electronics (diagnostic).
- **Signal strength**: the Bluetooth signal strength (diagnostic, disabled by default).

> The battery level and SoC temperature arrive in a separate Bluetooth advertisement from the cooking temperatures (food, ambient, and probe tip), so they can take a few seconds longer to appear after setup.

Reviewer: ```suggestion
The battery level and <abbr title="system-on-chip">SoC</abbr> temperature are sent in a separate Bluetooth advertisement from the cooking temperatures (food, ambient, and probe tip), so they may appear a few seconds later after setup.
```

---
title: Chef iQ
description: Instructions on how to integrate Chef iQ wireless probes with Home Assistant.
ha_category:
  - Sensor
ha_release: 2026.7
ha_iot_class: Local Push
ha_config_flow: true
ha_codeowners:
  - '@Invader444'
ha_domain: chefiq
ha_integration_type: device
ha_platforms:
  - diagnostics
  - sensor
ha_bluetooth: true
ha_quality_scale: bronze
---

The **Chef iQ** {% term integration %} lets you read the temperatures from Chef iQ
wireless cooking probes directly over Bluetooth Low Energy. No cloud account, base
station, or hub is required: the probe broadcasts its readings and Home Assistant

> listens passively.

Author: Addressed

{% include integrations/config_flow.md %}

## Supported devices

The following Chef iQ wireless probes are supported:

- Chef iQ Smart Wireless Meat Thermometer (CQ60)
- Chef iQ Smart Wireless Meat Thermometer (CQ50)

The **iQ Sense** base/hub broadcasts under the same Bluetooth identifier, but it is
a Wi-Fi appliance rather than a passive probe, so this integration does not set it

> up.

Author: Addressed

## Prerequisites

This integration requires a working [Bluetooth](/integrations/bluetooth/) setup,
either a local Bluetooth adapter or an [ESPHome Bluetooth proxy](/integrations/esphome/).
The probe is a sleepy device that only broadcasts while it is awake, so make sure it

> is awake (for example, by taking it off its charging dock) before adding it.

Author: Addressed

## Sensors

A probe exposes the following sensors:

| Sensor | Description |
| ------ | ----------- |
| Food temperature | The food (internal) temperature at the probe tip. |
| Ambient temperature | The ambient (cooking environment) temperature. |
| Probe tip 1-4 temperature | The temperatures measured along the probe shaft. |
| Battery | The probe battery percentage. |
| SoC temperature | The probe's internal electronics temperature (diagnostic). |

> | Signal strength | The Bluetooth signal strength (diagnostic, disabled by default). |

Author: Addressed

Battery and SoC temperature are published in a separate advertisement from the

> temperatures, so they can take a few seconds longer to appear after setup.

Author: Addressed

## Known limitations

The integration is passive and read-only. It reports the probe's broadcast
readings, but cannot start cooks, set target temperatures, or change probe

> settings. Those features require the Chef iQ app.

Author: Addressed
